### PR TITLE
reversing linked list in simple way..

### PR DESCRIPTION
**Pull request by tanmesh**

### CODE REVIEW

File: LL-reverse.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}node;
+
+void build(node** headRef) {
+	int n;
+	cout << "n\n";
+	cin >> n;
+	cout << "give data\n";
+	int x;
+	cin >> x;
+	node* newNode=(node*)malloc(sizeof(node));
```
Comment (nilmish) [inline LL-reverse.cpp]: abstract out a new function: `makeLinkedList`

File: LL-reverse.cpp [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	node* current=head;
+	while(current != NULL){
+		cout << current->data << " ";
+		current=current->next;
+	}
+	cout << endl;
+}
+
+void reverse(node** head) {
+	node* reverseHead=NULL;
```
Comment (nilmish) [inline LL-reverse.cpp]: better name for `reverseHead` would be  `previous`

Comment (tanmesh) [inline LL-reverse.cpp]: I mean... head of reverse LL

Comment (nilmish) [inline LL-reverse.cpp]: which would be prev at the end of the loop. the way you are using this variable is like its pointing to the previous node. At the end of the loop, this prev node happens to be the head of the reversed linked list as well.

**Comment by nilmish** [comment]
Replied.

**Pull request merged**